app/imports/components/apply/text-test.jsx

```diff
@@ -1,9 +1,32 @@
-import React from 'react';
+import React, { Component } from 'react';
 
-export default TextTest = () => (
-  <div class="content">
-    <h1>HTML Ipsum Presents</h1>
+// Import methods
+import { saveApplicationSection } from '/imports/api/methods.js';
+import { saveApplyPosition } from '/imports/api/methods.js';
 
-    <p><strong>Pellentesque habitant morbi tristique</strong> senectus et netus et malesuada fames ac turpis egestas. Vestibulum tortor quam, feugiat vitae, ultricies eget, tempor sit amet, ante. Donec eu libero sit amet quam egestas semper. <em>Aenean ultricies mi vitae est.</em> Mauris placerat eleifend leo. Quisque sit amet est et sapien ullamcorper pharetra. Vestibulum erat wisi, condimentum sed, <code>commodo vitae</code>, ornare sit amet, wisi. Aenean fermentum, elit eget tincidunt condimentum, eros ipsum rutrum orci, sagittis tempus lacus enim ac dui. <a href="#">Donec non enim</a> in turpis pulvinar facilisis. Ut felis.</p>
-  </div>
-);
+export default class TextTest extends Component {
+  savePosition() {
```

Comment:
I'm figuring out how to avoid this duplicated code:

Some help would be good: https://medium.com/@dan_abramov/mixins-are-dead-long-live-higher-order-components-94a0d2f9e750#.i153ibmci